.werf/stages/bundle.yaml

```diff
@@ -1,28 +1,30 @@
 # Bundle image, stored in your.registry.io/modules/<module-name>:<semver>
 ---
 image: bundle
-from: {{ .Images.BUILD_SCRATCH }}
+from: {{ .Images.BUILDER_SCRATCH }}
 import:
 # Rendering .werf/stages/images-digests.yaml is required! Comment out if not needed.
-- image: images-digests
-  add: /images_digests.json
-  to: /images_digests.json
-  after: setup
+- image: images-digests-and-version
+  add: /
+  to: /
+  includePaths:
+  - images_digests.json
+  - version.json
```

Comment:
Need to add `after: setup` here or werf's will fail with `Error: unable to load werf config: artifact stage is not specified with `before: install|setup` or `after: install|setup` for import!`